File: hazi/kovacs_norbert/butorgyar/Butor.java
package butorgyar;

class Butor {
    private Butorlap[] butorlapok;
    private int butorlapokSzama;

    public Butor() {
        butorlapok = new Butorlap[100]; 
        butorlapokSzama = 0;
    }

    public void lapHozzaad(Butorlap lap) {
        if (butorlapokSzama < butorlapok.length) {
            butorlapok[butorlapokSzama] = lap;
            butorlapokSzama++;
        }
    }

    public int arSzamol() {
        int teljesAr = 0;
        for (int i = 0; i < butorlapokSzama; i++) {
            teljesAr += butorlapok[i].arSzamol();
        }
        return teljesAr;
    }

    
    public String toString() {
        return "Butor - lapok: " + butorlapokSzama + ", ar " + arSzamol() + " Ft";
    }
}

File: python_kozepszint/Butorgyar/Butor.py
from typing import List
from Butorgyar.Butorlap import Butorlap as Butorlap 

class Butor:

    def __init__(self):
        self.butorlapok : List[Butorlap] = []
        self.butorlapokszama=0
    
    
    def lapHozzaad(self,butorlap:Butorlap):
        """
        parameter:
        butorlap: Butorlap
        """
        self.butorlapok.append(butorlap)
        self.butorlapokszama+=1

    def arSzamol(self):
        osszar=0
        for item in self.butorlapok:
            osszar+=item.arSzamol()
        return osszar
    
    def __str__(self):
        return f"Botor - lapok szama: {self.butorlapokszama} Ára: {self.arSzamol()} Ft"